toolchains/python/python.bzl

```
@@ -200,3 +200,184 @@ def nixpkgs_python_configure(
 
     if register:
         native.register_toolchains("@{}//:toolchain".format(name))
+
+
+def _nixpkgs_python_repository_impl(repository_ctx):
+    # 2. read generated json
+    python_modules = repository_ctx.read(repository_ctx.path(repository_ctx.attr.json_deps))
+
+    content = 'load("//:python_module.bzl", "python_module");'
```

Again related to the Q/A discussion. Since this file is the same every time it'd be better to include it in `rules_nixpkgs_python` as a source and just load it from there. 
```suggestion
    content = 'load("@rules_nixpkgs_python//:python_module.bzl", "python_module");'
```

---

Like an export_files, or a bzl_library ?

---

Shouldn't be needed for the `load`. At the moment `bzl_library` is only needed for Stardoc and this generated file will not have Stardoc generated for it. `export_files` is only needed if you want to make the file a source for a rule in another BUILD file.